In [1]:
import pandas as pd

In [2]:
ad=pd.read_csv("Adidas US Sales Datasets.csv")

In [3]:
ad.head(2)

Retailer  Retailer ID Invoice Date     Region     State      City  \
0  Foot Locker      1185732     1/1/2020  Northeast  New York  New York   
1  Foot Locker      1185732     1/2/2020  Northeast  New York  New York   

                   Product Price per Unit  Units Sold Total Sales  \
0    Men's Street Footwear        $50.00         1200   $600,000    
1  Men's Athletic Footwear        $50.00         1000   $500,000    

  Operating Profit Operating Margin Sales Method  
0        $300,000               50%     In-store  
1        $150,000               30%     In-store

In [4]:
for i in range(9648):
    ad['Operating Profit'][i]=ad['Operating Profit'][i].replace('$',"")
    ad['Operating Profit'][i]=ad['Operating Profit'][i].replace(',',"")
    ad['Operating Profit'][i]=ad['Operating Profit'][i].replace(' ',"")
    ad['Price per Unit'][i]=ad['Price per Unit'][i].replace('$',"")
    ad['Price per Unit'][i]=ad['Price per Unit'][i].replace(',',"")
    ad['Price per Unit'][i]=ad['Price per Unit'][i].replace(' ',"")
    ad['Price per Unit'][i]=ad['Price per Unit'][i].replace('.',"")
    ad['Total Sales'][i]=ad['Total Sales'][i].replace('$',"")
    ad['Total Sales'][i]=ad['Total Sales'][i].replace(',',"")
    ad['Total Sales'][i]=ad['Total Sales'][i].replace(' ',"")


/tmp/ipykernel_4305/614905253.py:2: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  ad['Operating Profit'][i]=ad['Operating Profit'][i].replace('$',"")
/tmp/ipykernel_4305/614905253.py:3: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  ad['Operating Profit'][i]=ad['Operating Profit'][i].replace(',',"")
/tmp/ipykernel_4305/614905253.py:4: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  ad['Operating Profit'][i]=ad['Operating Profit'][i].repla

In [10]:
ad['Price per Unit']=pd.to_numeric(ad["Price per Unit"])
ad['Total Sales']=pd.to_numeric(ad["Total Sales"])
ad['Operating Profit']=pd.to_numeric(ad["Operating Profit"])
for i in range (9648):
    ad['Price per Unit'][i]=ad['Price per Unit'][i]/100
    
    #ad['Operating Profit'][i]=int(ad['Operating Profit'][i]/10)
    #ad['Total Sales'][i]=int(ad['Total Sales'][i]/10)

/tmp/ipykernel_4305/784107697.py:5: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  ad['Price per Unit'][i]=ad['Price per Unit'][i]/100


In [11]:
ad.head(2)


Retailer  Retailer ID Invoice Date     Region     State      City  \
0  Foot Locker      1185732     1/1/2020  Northeast  New York  New York   
1  Foot Locker      1185732     1/2/2020  Northeast  New York  New York   

                   Product  Price per Unit  Units Sold  Total Sales  \
0    Men's Street Footwear              50        1200        60000   
1  Men's Athletic Footwear              50        1000        50000   

   Operating Profit Operating Margin Sales Method  
0             30000              50%     In-store  
1             15000              30%     In-store

In [12]:
ad.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 9648 entries, 0 to 9647
Data columns (total 13 columns):
 #   Column            Non-Null Count  Dtype 
---  ------            --------------  ----- 
 0   Retailer          9648 non-null   object
 1   Retailer ID       9648 non-null   int64 
 2   Invoice Date      9648 non-null   object
 3   Region            9648 non-null   object
 4   State             9648 non-null   object
 5   City              9648 non-null   object
 6   Product           9648 non-null   object
 7   Price per Unit    9648 non-null   int64 
 8   Units Sold        9648 non-null   int64 
 9   Total Sales       9648 non-null   int64 
 10  Operating Profit  9648 non-null   int64 
 11  Operating Margin  9648 non-null   object
 12  Sales Method      9648 non-null   object
dtypes: int64(5), object(8)
memory usage: 980.0+ KB


In [13]:
ad.describe()

Retailer ID  Price per Unit   Units Sold   Total Sales  \
count  9.648000e+03     9648.000000  9648.000000   9648.000000   
mean   1.173850e+06       45.216625   256.930037   9327.127384   
std    2.636038e+04       14.705397   214.252030  14191.735260   
min    1.128299e+06        7.000000     0.000000      0.000000   
25%    1.185732e+06       35.000000   106.000000    425.000000   
50%    1.185732e+06       45.000000   176.000000    957.000000   
75%    1.185732e+06       55.000000   350.000000  15000.000000   
max    1.197831e+06      110.000000  1275.000000  82500.000000   

       Operating Profit  
count       9648.000000  
mean        3442.202322  
std         5419.414426  
min            0.000000  
25%          192.000000  
50%          437.000000  
75%         5206.000000  
max        39000.000000

In [14]:
int(ad['Total Sales'].sum())

89988125

In [15]:
int(ad['Operating Profit'].sum())

33210368

In [16]:
int(ad['Total Sales'].sum()-ad['Operating Profit'].sum())

56777757

In [17]:
ad['Invoice Date'][0]

'1/1/2020'

In [18]:
ad['Invoice Date'][9647]

'2/22/2021'

In [19]:
366+31+22

419

In [20]:
ad.isna().sum()

Retailer            0
Retailer ID         0
Invoice Date        0
Region              0
State               0
City                0
Product             0
Price per Unit      0
Units Sold          0
Total Sales         0
Operating Profit    0
Operating Margin    0
Sales Method        0
dtype: int64

In [21]:
ad['Product'].value_counts()

Men's Street Footwear        1610
Men's Athletic Footwear      1610
Women's Street Footwear      1608
Women's Apparel              1608
Women's Athletic Footwear    1606
Men's Apparel                1606
Name: Product, dtype: int64

In [22]:
#total unit sold
p=ad['Product'].unique()
s=0
e=[]
for i in range(len(p)):
    for j in range(9648):
        if(ad['Product'][j]==p[i]):
            s=s+ad['Units Sold'][j]
    e.append(s)
    print(p[i]," - ",s)
    s=0

Men's Street Footwear  -  593320
Men's Athletic Footwear  -  435526
Women's Street Footwear  -  392269
Women's Athletic Footwear  -  317236
Men's Apparel  -  306683
Women's Apparel  -  433827


In [23]:
#total sales
p=ad['Product'].unique()
s=0
ps=[]
for i in range(len(p)):
    for j in range(9648):
        if(ad['Product'][j]==p[i]):
            s=s+ad['Total Sales'][j]
    ps.append(s)
    print(p[i]," - ",int(s))
    s=0

Men's Street Footwear  -  20882270
Men's Athletic Footwear  -  15367024
Women's Street Footwear  -  12799931
Women's Athletic Footwear  -  10662832
Men's Apparel  -  12372522
Women's Apparel  -  17903546


In [24]:
Re=ad['Region'].unique()
s=0
Res=[]
for i in range(len(Re)):
    for j in range(9648):
        if(ad['Region'][j]==Re[i]):
            s=s+ad['Total Sales'][j]
    Res.append(s)
    print(Re[i]," - ",int(s))
    s=0

Northeast  -  18631879
South  -  14465919
West  -  26993774
Midwest  -  13579676
Southeast  -  16316877


In [25]:
R=ad['Retailer'].unique()
s=0
Rs=[]
for i in range(len(R)):
    for j in range(9648):
        if(ad['Retailer'][j]==R[i]):
            s=s+ad['Total Sales'][j]
    Rs.append(s)
    print(R[i]," - ",int(s))
    s=0

Foot Locker  -  22008905
Walmart  -  7455716
Sports Direct  -  18246638
West Gear  -  24295922
Kohl's  -  10211252
Amazon  -  7769692


In [26]:
Sm=ad['Sales Method'].unique()
s=0
k=0
Sms=[]
for i in range(len(Sm)):
    for j in range(9648):
        if(ad['Sales Method'][j]==Sm[i]):
            s=s+ad['Units Sold'][j]
            k=k+ad['Total Sales'][j]
    Sms.append(k)
    print(Sm[i]," - ",s," => ",k)
    s=0
    k=0

In-store  -  689990  =>  35664375
Outlet  -  849778  =>  29557920
Online  -  939093  =>  24765830


In [28]:
S=ad['State'].unique()
s=0
c=[]
for i in range(len(S)):
    for j in range(9648):
        if(ad['State'][j]==S[i]):
            s=s+ad['Total Sales'][j]
    print(S[i]," - ",int(s))
    s=0

New York  -  6422829
Texas  -  4635871
California  -  6017309
Illinois  -  979725
Pennsylvania  -  1035698
Nevada  -  2085803
Colorado  -  2099627
Washington  -  2633047
Florida  -  5928300
Minnesota  -  737776
Montana  -  1571062
Tennessee  -  1806696
Nebraska  -  592873
Alabama  -  1763290
Maine  -  919574
Alaska  -  1475280
Hawaii  -  2228221
Wyoming  -  1857727
Virginia  -  2157456
Michigan  -  1862512
Missouri  -  968300
Utah  -  969586
Oregon  -  2134917
Louisiana  -  2375033
Idaho  -  1927637
Arizona  -  1578167
New Mexico  -  1986453
Georgia  -  1899699
South Carolina  -  2928538
North Carolina  -  2395626
Ohio  -  1848428
Kentucky  -  1007258
Mississippi  -  1559119
Arkansas  -  1263889
Oklahoma  -  1062021
Kansas  -  997262
South Dakota  -  849528
North Dakota  -  773534
Iowa  -  742372
Wisconsin  -  772715
Indiana  -  883589
West Virginia  -  1068889
Maryland  -  775777
Delaware  -  1229816
New Jersey  -  993394
Connecticut  -  1157292
Rhode Island  -  844630
Massachusetts  

In [29]:
C=ad['City'].unique()
s=0
Cs=[]
for i in range(len(C)):
    for j in range(9648):
        if(ad['City'][j]==C[i]):
            s=s+ad['Total Sales'][j]
    Cs.append(s)
    print(C[i]," - ",s)
    s=0

New York  -  3980077
Houston  -  2545639
San Francisco  -  3453875
Los Angeles  -  2563434
Chicago  -  979725
Dallas  -  2090232
Philadelphia  -  1035698
Las Vegas  -  2085803
Denver  -  2099627
Seattle  -  2633047
Miami  -  3160061
Minneapolis  -  737776
Billings  -  1571062
Knoxville  -  1806696
Omaha  -  592873
Birmingham  -  1763290
Portland  -  3054491
Anchorage  -  1475280
Honolulu  -  2228221
Orlando  -  2768239
Albany  -  2442752
Cheyenne  -  1857727
Richmond  -  2157456
Detroit  -  1862512
St. Louis  -  968300
Salt Lake City  -  969586
New Orleans  -  2375033
Boise  -  1927637
Phoenix  -  1578167
Albuquerque  -  1986453
Atlanta  -  1899699
Charleston  -  3997427
Charlotte  -  2395626
Columbus  -  1848428
Louisville  -  1007258
Jackson  -  1559119
Little Rock  -  1263889
Oklahoma City  -  1062021
Wichita  -  997262
Sioux Falls  -  849528
Fargo  -  773534
Des Moines  -  742372
Milwaukee  -  772715
Indianapolis  -  883589
Baltimore  -  775777
Wilmington  -  1229816
Newark  -  993

In [30]:
p=ad['Product'].unique()
s=0
l=0
for i in range(len(p)):
    for j in range(9648):
        if(ad['Product'][j]==p[i]):
            s=s+ad['Price per Unit'][j]
            l+=1
    print("average cost per item ",p[i]," - ",int(s/l))
    s=0
    l=0

average cost per item  Men's Street Footwear  -  44
average cost per item  Men's Athletic Footwear  -  43
average cost per item  Women's Street Footwear  -  40
average cost per item  Women's Athletic Footwear  -  41
average cost per item  Men's Apparel  -  50
average cost per item  Women's Apparel  -  51


In [31]:
#average prepare cost per item
p=ad['Product'].unique()
s=0
for i in range(len(p)):
    for j in range(9648):
        if(ad['Product'][j]==p[i]):
            s=s+ad['Total Sales'][j]-ad['Operating Profit'][j]
            l+=1
    s=s/e[i]
    print(p[i]," - ",int(s))
    s=0


Men's Street Footwear  -  21
Men's Athletic Footwear  -  23
Women's Street Footwear  -  21
Women's Athletic Footwear  -  21
Men's Apparel  -  25
Women's Apparel  -  25


In [32]:
# average profit cost per item
p=ad['Product'].unique()
s=0
l=0
for i in range(len(p)):
    for j in range(9648):
        if(ad['Product'][j]==p[i]):
            s=s+ad['Operating Profit'][j]
    s=s/e[i]
    print("average operating profit ",p[i]," - ",int(s))
    s=0

average operating profit  Men's Street Footwear  -  13
average operating profit  Men's Athletic Footwear  -  11
average operating profit  Women's Street Footwear  -  11
average operating profit  Women's Athletic Footwear  -  12
average operating profit  Men's Apparel  -  14
average operating profit  Women's Apparel  -  15


In [220]:
Sm=ad['Sales Method'].unique()
s=0
z=0
for i in range(len(Sm)):
    print(Sm[i],"\n")
    for j in range(len(R)):
        for k in range(9648):
            if(ad['Retailer'][k]==R[j] and ad['Sales Method'][k]==Sm[i]):
                s=s+ad['Total Sales'][k]
                z=z+1
        print(R[j]," = ",z,"  ->  ",int(s))
        s=0
        z=0
    print("\n")
    
    
                

In-store 

Foot Locker  =  449   ->   7652500
Walmart  =  37   ->   1641125
Sports Direct  =  293   ->   5504850
West Gear  =  699   ->   15672650
Kohl's  =  144   ->   2956625
Amazon  =  118   ->   2236625


Outlet 

Foot Locker  =  793   ->   7062490
Walmart  =  301   ->   4307708
Sports Direct  =  744   ->   6819484
West Gear  =  580   ->   4570422
Kohl's  =  310   ->   4155569
Amazon  =  291   ->   2642247


Online 

Foot Locker  =  1395   ->   7293915
Walmart  =  288   ->   1506883
Sports Direct  =  995   ->   5922304
West Gear  =  1095   ->   4052850
Kohl's  =  576   ->   3099058
Amazon  =  540   ->   2890820




In [217]:
p=ad['Product'].unique()
s=0
z=0
for i in range(len(R)):
    print(R[i],"\n")
    for j in range(len(p)):
        for k in range(9648):
            if(ad['Product'][k]==p[j] and ad['Retailer'][k]==R[i]):
                s=s+1
                z=z+ad['Total Sales'][k]
        print(p[j]," = ",int(s),"  ->  ",int(z))
        s=0
        z=0
    print("\n")
    
                

Foot Locker 

Men's Street Footwear  =  449   ->   5748059
Men's Athletic Footwear  =  442   ->   3647952
Women's Street Footwear  =  438   ->   2908704
Women's Athletic Footwear  =  442   ->   2423858
Men's Apparel  =  433   ->   2950810
Women's Apparel  =  433   ->   4329522


Walmart 

Men's Street Footwear  =  101   ->   1492403
Men's Athletic Footwear  =  104   ->   1360017
Women's Street Footwear  =  99   ->   1046398
Women's Athletic Footwear  =  102   ->   876262
Men's Apparel  =  113   ->   1149044
Women's Apparel  =  107   ->   1531592


Sports Direct 

Men's Street Footwear  =  335   ->   3867153
Men's Athletic Footwear  =  337   ->   3088896
Women's Street Footwear  =  342   ->   2782442
Women's Athletic Footwear  =  338   ->   2314445
Men's Apparel  =  339   ->   2482453
Women's Apparel  =  341   ->   3711249


West Gear 

Men's Street Footwear  =  394   ->   5316457
Men's Athletic Footwear  =  396   ->   4280498
Women's Street Footwear  =  400   ->   3638569
Women's Athle

In [249]:
print("mean = ",int(ad['Total Sales'].mean()),"\nmin = ",
int(ad['Total Sales'].min()),"\nmax = ",
int(ad['Total Sales'].max()),"\ntotal sales = ",
int(ad['Total Sales'].sum()),"\nstandard deviation = ",
int(ad['Total Sales'].std()))

mean =  9327 
min =  0 
max =  82500 
total sales =  89988125 
standard deviation =  14191


In [253]:
mi=ad['Total Sales'].min()
ad.loc[ad['Total Sales']==mi]
#no sale products

Retailer  Retailer ID Invoice Date   Region     State   City  \
1019  Foot Locker      1185732     6/5/2021  Midwest  Nebraska  Omaha   
1025  Foot Locker      1185732    6/11/2021  Midwest  Nebraska  Omaha   
4907  Foot Locker      1185732     6/5/2021  Midwest  Nebraska  Omaha   
4913  Foot Locker      1185732    6/11/2021  Midwest  Nebraska  Omaha   

                        Product  Price per Unit  Units Sold  Total Sales  \
1019  Women's Athletic Footwear              35           0          0.0   
1025  Women's Athletic Footwear              30           0          0.0   
4907  Women's Athletic Footwear              33           0          0.0   
4913  Women's Athletic Footwear              27           0          0.0   

      Operating Profit Operating Margin Sales Method  
1019               0.0              40%       Outlet  
1025               0.0              40%       Outlet  
4907               0.0              55%       Online  
4913               0.0              53%       Online

In [272]:
ad.loc[ad['Total Sales']==ad['Total Sales'].max()]
#high sale products

Retailer  Retailer ID Invoice Date     Region      State         City  \
1394    Walmart      1128299    6/17/2021  Southeast    Florida      Orlando   
1970  West Gear      1128299    6/17/2021      South  Louisiana  New Orleans   

              Product  Price per Unit  Units Sold  Total Sales  \
1394  Women's Apparel             110         750      82500.0   
1970  Women's Apparel             110         750      82500.0   

      Operating Profit Operating Margin Sales Method  
1394           37125.0              45%     In-store  
1970           28875.0              35%       Online

In [274]:
ad.loc[ad['Price per Unit']==ad['Price per Unit'].max()]#costly sale products
#costly products

Retailer  Retailer ID Invoice Date     Region      State         City  \
1394    Walmart      1128299    6/17/2021  Southeast    Florida      Orlando   
1400    Walmart      1128299    7/16/2021  Southeast    Florida      Orlando   
1970  West Gear      1128299    6/17/2021      South  Louisiana  New Orleans   
1976  West Gear      1128299    7/16/2021      South  Louisiana  New Orleans   

              Product  Price per Unit  Units Sold  Total Sales  \
1394  Women's Apparel             110         750      82500.0   
1400  Women's Apparel             110         650      71500.0   
1970  Women's Apparel             110         750      82500.0   
1976  Women's Apparel             110         650      71500.0   

      Operating Profit Operating Margin Sales Method  
1394           37125.0              45%     In-store  
1400           32175.0              45%     In-store  
1970           28875.0              35%       Online  
1976           25025.0              35%       Online

In [275]:
ad.loc[ad['Price per Unit']==ad['Price per Unit'].min()]
#cheapest products

Retailer  Retailer ID Invoice Date Region     State  \
8372  Sports Direct      1197831    8/25/2021  South   Alabama   
8609      West Gear      1185732    9/12/2021   West      Utah   
8610      West Gear      1185732    9/12/2021   West      Utah   
9221      West Gear      1197831    3/19/2021  South  Oklahoma   

                City                    Product  Price per Unit  Units Sold  \
8372      Birmingham      Men's Street Footwear               7         224   
8609  Salt Lake City    Women's Street Footwear               7         105   
8610  Salt Lake City  Women's Athletic Footwear               7         111   
9221   Oklahoma City    Women's Street Footwear               7          83   

      Total Sales  Operating Profit Operating Margin Sales Method  
8372        156.0              73.0              47%       Outlet  
8609         73.0              30.0              42%       Outlet  
8610         77.0              35.0              46%       Outlet  
9221         58.0              26.0              45%       Outlet

In [279]:
print("maximum sales :\n")
print("product")
for i in range(len(p)):
    if(ps[i]==max(ps)):
        print(p[i]," -> ",ps[i],"\n")

print("retailer")
for i in range(len(p)):
    if(Rs[i]==max(Rs)):
        print(R[i]," -> ",Rs[i],"\n")

print("Sales method")
for i in range(len(Sm)):
    if(Sms[i]==max(Sms)):
        print(Sm[i]," -> ",Sms[i],"\n")
        
print("City")
for i in range(len(Cs)):
    if(Cs[i]==max(Cs)):
        print(C[i]," -> ",Cs[i],"\n")

print("Region")
for i in range(len(Res)):
    if(Res[i]==max(Res)):
        print(Re[i]," -> ",Res[i],"\n")

maximum sales :

product
Men's Street Footwear  ->  20882270.0 

retailer
West Gear  ->  24295922.0 

Sales method
In-store  ->  35664375.0 

City
Charleston  ->  3997427.0 

Region
West  ->  26993774.0 



In [231]:
ad.head(11)

Retailer  Retailer ID Invoice Date     Region     State      City  \
0   Foot Locker      1185732     1/1/2020  Northeast  New York  New York   
1   Foot Locker      1185732     1/2/2020  Northeast  New York  New York   
2   Foot Locker      1185732     1/3/2020  Northeast  New York  New York   
3   Foot Locker      1185732     1/4/2020  Northeast  New York  New York   
4   Foot Locker      1185732     1/5/2020  Northeast  New York  New York   
5   Foot Locker      1185732     1/6/2020  Northeast  New York  New York   
6   Foot Locker      1185732     1/7/2020  Northeast  New York  New York   
7   Foot Locker      1185732     1/8/2020  Northeast  New York  New York   
8   Foot Locker      1185732    1/21/2020  Northeast  New York  New York   
9   Foot Locker      1185732    1/22/2020  Northeast  New York  New York   
10  Foot Locker      1185732    1/23/2020  Northeast  New York  New York   

                      Product  Price per Unit  Units Sold  Total Sales  \
0       Men's Street Footwear              50        1200      60000.0   
1     Men's Athletic Footwear              50        1000      50000.0   
2     Women's Street Footwear              40        1000      40000.0   
3   Women's Athletic Footwear              45         850      38250.0   
4               Men's Apparel              60         900      54000.0   
5             Women's Apparel              50        1000      50000.0   
6       Men's Street Footwear              50        1250      62500.0   
7     Men's Athletic Footwear              50         900      45000.0   
8     Women's Street Footwear              40         950      38000.0   
9   Women's Athletic Footwear              45         825      37125.0   
10              Men's Apparel              60         900      54000.0   

    Operating Profit Operating Margin Sales Method  
0            30000.0              50%     In-store  
1            15000.0              30%     In-store  
2            14000.0              35%     In-store  
3            13387.0              35%     In-store  
4            16200.0              30%     In-store  
5            12500.0              25%     In-store  
6            31250.0              50%     In-store  
7            13500.0              30%       Outlet  
8            13300.0              35%       Outlet  
9            12993.0              35%       Outlet  
10           16200.0              30%       Outlet

total days=419

total sales= 89988125

total profit= 33210368

maximum sales :

product
Men's Street Footwear  ->  20882270.0 

retailer
West Gear  ->  24295922.0 

Sales method
In-store  ->  35664375.0 

City
Charleston  ->  3997427.0 

Region
West  ->  26993774.0 

In [36]:
s=0
z=0
for l in range(len(Sm)):
    print(Sm[l],"\n")
    for i in range(len(R)):
        print(R[i],"\n")
        for j in range(len(p)):
            for k in range(9648):
                if(ad['Product'][k]==p[j] and ad['Retailer'][k]==R[i] and ad['Sales Method'][k]==Sm[l]):
                    s=s+1
                    z=z+ad['Total Sales'][k]
            print(p[j]," = ",int(s),"  ->  ",int(z))
        s=0
        z=0
    print("\n")
    
                

In-store 

Foot Locker 

Men's Street Footwear  =  77   ->   2087275
Men's Athletic Footwear  =  151   ->   3333525
Women's Street Footwear  =  223   ->   4253875
Women's Athletic Footwear  =  297   ->   5015375
Men's Apparel  =  373   ->   6090625
Women's Apparel  =  449   ->   7652500
Walmart 

Men's Street Footwear  =  6   ->   306125
Men's Athletic Footwear  =  13   ->   676375
Women's Street Footwear  =  19   ->   929250
Women's Athletic Footwear  =  25   ->   1138250
Men's Apparel  =  31   ->   1363125
Women's Apparel  =  37   ->   1641125
Sports Direct 

Men's Street Footwear  =  48   ->   1480950
Men's Athletic Footwear  =  97   ->   2325575
Women's Street Footwear  =  147   ->   3033850
Women's Athletic Footwear  =  196   ->   3660100
Men's Apparel  =  245   ->   4406350
Women's Apparel  =  293   ->   5504850
West Gear 

Men's Street Footwear  =  115   ->   3584075
Men's Athletic Footwear  =  231   ->   6280325
Women's Street Footwear  =  350   ->   8591150
Women's Athletic Fo